### PR TITLE
Packages cleanup and config improvements

### PR DESCRIPTION
**Pull request by ValeriaNARDET**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,34 +21,22 @@
...
     "lodash": "4.17.11",
     "moment": "2.23.0",
-    "node-polyglot": "2.3.0",
     "numeral": "2.0.6",
-    "tokend-js-sdk": "git+https://github.com/tokend/js-sdk.git",
-    "ua-parser-js": "0.7.19",
-    "vee-validate": "2.1.5",
     "vue": "2.5.21",
     "vue-flatpickr-component": "8.1.0",
-    "vue-markdown": "2.2.4",
```
Comment (max-nes) [inline package.json]: please update the branch to the latest master, vue-markdown is being used already

File: package.json [hunk 2]
```diff
@@ -113,7 +98,6 @@
     "vue-loader": "15.4.2",
     "vue-template-compiler": "2.5.21",
     "webpack": "4.28.3",
-    "webpack-bundle-analyzer": "3.0.3",
```
Comment (max-nes) [inline package.json]: I removed it by mistake, please leave it. Is convenient tool to analyze bundle sizes

File: src/config.js [hunk 3]
```diff
@@ -1,45 +1,60 @@
...
+     * sipped step, the user will be instantly logged in and redirected
+     * to the application.
+     */
+    SKIP_EMAIL_CONFIRMATION_STEP: false,
+
+    /**
+     * Disables certain features of the application. Set a property to false to
+     * restrict usage of the feature by all users of the system
+     */
+    FEATURE_FLAGS: {
```
Comment (max-nes) [inline src/config.js]: Feature flags could also be updated after you update the branch to the latest master

File: src/config.js [hunk 4]
```diff
@@ -1,45 +1,60 @@
...
-    NULL_ASSET_SIGNER: 'GAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHV4',
-    LOCAL_STORAGE_KEY: 'tokend-client',
-    DEFAULT_QUOTE_ASSET: 'USD',
-    DEFAULT_TRADE_PAIRS_RE: [
-      // Descending priority
-      /BTC.*\/.*ETH/,
-      /ETH.*\/.*BTC/,
-    ],
+
+    // TODO: remove
```
Comment (max-nes) [inline src/config.js]: resolve this todo please if `DEBUG` is not being used anymore

File: src/vue/common/EmailGetter.spec.js [hunk 5]
```diff
@@ -13,6 +13,7 @@ Vue.config.silent = true
 
 const localVue = createLocalVue()
 
+// TODO: tests
```
Comment (max-nes) [inline src/vue/common/EmailGetter.spec.js]: resolve this too please

Comment (ValeriaNARDET) [reply src/vue/common/EmailGetter.spec.js]: Should I run tests or shuld I look some action at site ?

File: src/config.js (hunk 3 above)
Comment (ValeriaNARDET) [reply src/config.js]: Yes, it was renamed as 'featureFlags' in config but is used as 'FEATURE_FLAGS' in router and sidebar on master

File: src/vue/common/EmailGetter.spec.js (hunk 5 above)
Comment (max-nes) [reply src/vue/common/EmailGetter.spec.js]: remove the comment so

File: package.json [hunk 6]
```diff
@@ -21,34 +21,24 @@
...
-    "tokend-js-sdk": "git+https://github.com/tokend/js-sdk.git",
-    "ua-parser-js": "0.7.19",
-    "vee-validate": "2.1.5",
     "vue": "2.5.21",
     "vue-flatpickr-component": "8.1.0",
     "vue-markdown": "2.2.4",
-    "vue-material": "1.0.0-beta-7",
     "vue-qr": "1.5.2",
-    "vue-qrcode-component": "2.1.1",
-    "vue-resource": "1.5.1",
```
Comment (ardier16) [inline package.json]: Please, keep `vue-resource` until that TODO is resolved. @max-nes @ValeriaNARDET @quinn71 

https://github.com/tokend/web-client/blob/65402043f10538fe84bab77d4a3df8facc522efa/src/js/helpers/document-uploader.js#L71-L72

**Pull request merged**